MAUI/Maps/tile-layer.md

```diff
@@ -213,4 +213,43 @@ tileLayer.DeleteTilesFromCache();
 
 You can add markers in the tile layer. The procedure is very similar to the shape layer. Kindly refer to the [markers](https://help.syncfusion.com/maui/maps/markers) section.
 
-N> You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations. You can also explore our [`.NET MAUI Maps Tile layer example`](https://github.com/syncfusion/maui-demos/) which shows how to configure a Maps in .NET MAUI.
\ No newline at end of file
+N> You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations. You can also explore our [`.NET MAUI Maps Tile layer example`](https://github.com/syncfusion/maui-demos/) which shows how to configure a Maps in .NET MAUI.
+
+##  Event
+
+The [`CenterChanged`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_CenterChanged) event is triggered while zooming and panning the maps.
+
+The following arguments can be obtained from the `MapTileLayer_CenterChanged` event: 
```

Comment:
The following arguments can be obtained from the `MapTileLayer_CenterChanged` event.

Reply:
Content modified.